README.rst

```diff
@@ -71,15 +71,15 @@ Example
 
         async def update_stats(self):
             """This function runs every 30 minutes to automatically update your server count"""
-           await self.bot.is_ready()
-            while not bot.is_closed:
```

Comment:
This is so that when the bot closes cleaning so does the loop, a while True loop will hang the bot when it closes